akka/src/main/scala/anorm/Done.scala

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (C) 2016 Lightbend Inc. <http://www.lightbend.com>
+ */
+package anorm
+
+/**
+ * Typically used together with `Future` to signal completion
+ * but there is no actual value completed. More clearly signals intent
+ * than `Unit` and is available both from Scala and Java (which `Unit` is not).
+ */
+sealed abstract class Done
+
+case object Done extends Done {
```

Comment:
Akka [`Done`](http://doc.akka.io/api/akka/2.4.8/?_ga=1.256176713.331620270.1455976396#akka.Done)  is available